frontend/components/AddSite/TemplateSiteList/templateSite.js

```diff
@@ -55,59 +55,56 @@ class TemplateSite extends React.Component {
+            className="thumbnail"
+            src={this.props.thumb}
+            alt={`Thumbnail screenshot for the ${this.props.title} template`}
+          />
+          <h3 className="well-title">{this.props.title}</h3>
+        </div>
+        <div className="well-text">
+          <p>{this.props.description}</p>
+          {this.getFormVisible() ?
+            <form
+              className="new-site-form"
+              onSubmit={this.handleSubmit}
+            >
+              <label htmlFor="repository">What GitHub account will own your site?</label>
+              <input
+                name="owner"
+                type="text"
+                value={this.state.owner}
+                onChange={this.handleChange}
+              />
+              <label htmlFor="repository">Name your new site</label>
+              <input
+                name="repository"
+                type="text"
+                value={this.state.repository}
+                onChange={this.handleChange}
               />
-            </a>
-            <p>{this.props.description}</p>
-            <div className="button_wrapper">
-              <a
-                className="usa-button usa-button-outline"
-                href={this.props.example}
-                target="_blank"
-                rel="noopener noreferrer"
-                role="button"
-              >
-                Example
-              </a>
-              <button
-                className="usa-button"
-                onClick={this.handleChooseActive}
-              >
-                Use this template
-              </button>
-            </div>
-            {this.getFormVisible() ?
-              <form
-                className="new-site-form"
-                onSubmit={this.handleSubmit}
-              >
-                <label htmlFor="repository">What GitHub account will own your site?</label>
-                <input
-                  name="owner"
-                  type="text"
-                  value={this.state.owner}
-                  onChange={this.handleChange}
-                />
-                <label htmlFor="repository">Name your new site</label>
-                <input
-                  name="repository"
-                  type="text"
-                  value={this.state.repository}
-                  onChange={this.handleChange}
-                />
-                <input type="submit" value="Create site" />
-              </form>
-            : null}
-          </div>
+              <input type="submit" value="Create site" />
+            </form>
+          : null}
+        </div>
+        <div className="well-footer">
+          <a
+            href={this.props.example}
+            target="_blank"
+            rel="noopener noreferrer"
+            role="button"
+          >
+            View template
+          </a>
+          <button
+            className="usa-button"
+            onClick={this.handleChooseActive}
+          >
+            Use this template
```

Comment:
hrm, looks like this isn't working properly any more.

Upon expansion, it should change to a "Create Site" button, but now it is keeping the "View Template" and "Use this template" link.

Reply:
Also when you expand one, they all expand:

![expand-problem](https://user-images.githubusercontent.com/697848/33077258-f9cc7540-ce94-11e7-8223-54315a5cf1ef.gif)

Reply:
This is the screenshot from the live site where the example and use this template buttons don't go away when clicking create site.  I remember thinking this was odd and also thought it was odd that there was no close button. I'd def like to revisit the UX/UI for the use this template flow.  Regarding all of them expanding, that was in the original request. Make all the template wells the same height 😆 

![screen shot 2017-11-21 at 9 42 34 am](https://user-images.githubusercontent.com/1449852/33078196-e92f76a0-ce9f-11e7-9390-6cc0a53c1232.png)

CC @tram @OpenGlobe

Reply:
whoa, I totally missed that the original buttons were still present upon expansion in the current code o_O

Reply:
I'm unsure - how to fix the "use this template" expand/collapse. If that's a blocker to merging @jseppi I'll need to pair with you to get it sorted. Otherwise I think this PR is ready to merge

Reply:
I can do it. How is this for a first pass? We can iterate later on adding a cancel/close button:

![hide-use-buttons](https://user-images.githubusercontent.com/697848/33080115-e89a9d3a-ce9c-11e7-95d0-2f299b707995.gif)

Reply:
Yeah - I think that works! Thanks!